### DISCUSSION TITLE
how to add to project

### DISCUSSION THREAD

**Discussion by Mati4566456** [zompi2/UE4EasyLocalizationTool#11] [General]
how to add to project?

**Comment by zompi2** [comment]
Hi!

In order to add the plugin to the project you must:

1. Download all the repository to the `Plugins` directory of your project.
2. Add  "EasyLocalizationTool" to the `PublicDependencyModuleNames` in your project `.Build.cs`
3. Enable the plugin either via Editor or via modifying `.uproject` file

Unfortunatelly there are no binary builds for this plugin and it must be built from source.

You can check the example project: https://github.com/zompi2/UE4EasyLocalizationToolExample and see how it is integrated there.
To clone this repository with the plugin use command:
`git clone --recursive https://github.com/zompi2/UE4EasyLocalizationToolExample.git .`
